far2l/bootstrap/trash.sh

```diff
@@ -10,6 +10,6 @@ set -e
 
 if [ -x ~/.config/far2l/trash.sh ]; then
 . ~/.config/far2l/trash.sh
+else
+  gvfs-trash -f "$1" || gio trash -f "$1"
```

Comment:
using command -v would be 'nicer', see as example: https://github.com/elfmz/far2l/blob/master/far2l/bootstrap/notify.sh